src/storage/src/table/storage_table.rs

```diff
@@ -525,66 +541,16 @@ impl<S: StateStore, E: Encoding, const T: AccessType> StorageTableBase<S, E, T>
         Ok(iter)
     }
 
-    /// Get a [`StorageTableIter`] for streaming use with given `encoded_key_range`.
-    pub(super) async fn streaming_iter_with_encoded_key_range<R, B>(
-        &self,
-        encoded_key_range: R,
-        epoch: u64,
-    ) -> StorageResult<StorageTableIter<S>>
-    where
-        R: RangeBounds<B> + Send + Clone,
-        B: AsRef<[u8]> + Send,
-    {
-        self.iter_with_encoded_key_range(encoded_key_range, epoch, false, true)
-            .await
-    }
-
-    /// Get a [`StorageTableIter`] with given `encoded_key_range`.
-    /// Differs from the streaming one, this iterator will wait for the epoch before iteration.
-    pub(super) async fn batch_iter_with_encoded_key_range<R, B>(
+    /// Iterates on the table with the given prefix of the pk in `pk_prefix` and the range bounds of
+    /// the next primary key column in `next_col_bounds`.
+    // TODO: support multiple datums or `Row` for `next_col_bounds`.
```

Comment:
Should we support multiple datums in `next_col_bounds` here? So that all cases can reuse this function? 🤣 cc @xxchan

Reply:
I think other cases are simple. Reusing this function does no benefit to them 🥸

Reply:
But you already removed `iter_with_pk_prefix`... 🤔

Reply:
So when will `next_col_bounds` for multiple columns needed?

Reply:
If `next_col_bounds` can be multiple columns, then all of the scan patterns can be formalized by this method, including arbitrary pk bound scan.